core/menu_module.js

```diff
@@ -106,30 +103,25 @@ exports.MenuModule = class MenuModule extends PluginModule {
                         return callback(Errors.MissingConfig('Prompt specified but no "promptConfig" block found'));
                     }
 
+                    const options = self.menuConfig.config;
+
+                    if(artData != null && artData.height != null) {
```

Comment:
You should be able to do something like:
`if (_.isNumber(artData?.height)) { ... }`

Reply:
Sorry more of my old school js slipping through 🙂 thanks I'll update this

Reply:
Actually - as it turns out conditional chaining would require an upgrade to the `package.json` to upgrade eslint, and adding `"ecmaVersion": 2020` to the `.eslintrc.json`

I can do that, but it would also up the minimum version of Node.js as Node 12 doesn't support conditional chaining (looks like the minimum version for this is Node 14.) Are you ok with upping the minimum version?

Reply:
Yep let's do it!

Reply:
I went ahead and added the more verbose option for now. Let me know if you want me to bump the Node version instead.

Reply:
Ok I'll upgrade the minimum version as well to Node 14

Reply:
I bumped up the minimum version of Node to 14, and changed the ECMAScript version to 2020.

I also made an assumption and went ahead and bumped the enigma-bbs version to 0.0.14-beta. I figured a new Node version plus removing CPR probably deserves a version bump if you agree?

Reply:
Agree 100% that we should bump the enig version. We're at 0.0.12, so we can move to 0.0.13-beta now though.